homeassistant/components/sensor/icons.json

```diff
@@ -149,6 +149,9 @@
     },
     "wind_speed": {
       "default": "mdi:weather-windy"
+    },
+    "conductivity": {
```

Comment:
Please make a separate PR for this change. When making that PR, please also add an icon for the number integration for this device class. We missed this when adding this device class a couple of months ago.

Reply:
Please remove this change.